### DISCUSSION TITLE
Paths

### DISCUSSION THREAD

**Discussion by kinlane** [postman-open-technologies/linting-rules#23] [General]
This is a discussion around the path ruleset, keeping an ongoing record of what has gone into the ruleset.

**Comment by jwbanning** [comment]
From https://rules.linting.org/, The "Paths Trailing Slash - (paths-trailing-slash)" states:

 "message: Path with trailing slash is not allowed" 
but the description is
"description" : All paths must end with a trailing slash.

<img width="1129" alt="Screen Shot 2021-11-30 at 7 09 57 PM" src="https://user-images.githubusercontent.com/2584258/144153600-628a2bb9-f8b7-4c4e-80c9-8396f2de3f0f.png">

Same for the linting rule here:
https://rules.linting.org//rules/paths-trailing-slash/
